### DISCUSSION TITLE
v3.3.0

### DISCUSSION THREAD

**Discussion by Daveiano** [Daveiano/weewx-wdc#182] [Announcements]
- Bugfix: Some observations with aggregate_imterval `sum` could be updated to `NaN` via MQTT under some circumstances GH-166
- Bugfix: Getting the chart properties if observation is not included in [diagrams] config GH-172
- Bugfix: Minor Spelling Error on Some Tiles GH-168
- Added/fixed some more translations (Chart legends, Stats page)
- Bugfix: (MQTT) Wind Speed Direction and Wind Gust Direction are Equal/Change At Same Time GH-177
- Support for weewx-cmon GH-169 (https://github.com/Daveiano/weewx-wdc/wiki/Support-for-weewx-cmon)
- Add support for Classic layout on Sensor Status page
- Added support for displaying observations in stat tiles (and batteries) that are not saved to DB (only available via the `$current` tag) GH-140 and https://groups.google.com/g/weewx-user/c/AwhvlJ-DRUQ/m/bzLWZV19DQAJ
- Add support for weewx-xaggs (configurable tiles to show historical data) GH-170 (https://github.com/Daveiano/weewx-wdc/wiki/Support-for-weewx-xaggs)
- Extend Stats/Climatological days table. Added rain stats (last rain, most consecutive days of rain) GH-171
- New Chart Type: Gauge GH-155 (https://github.com/Daveiano/weewx-wdc/wiki/Gauges)
- Added support for custom XTypes (for charts) that need a \*\*config_dict GH-180
- Extended Webcams/Externals (see the updated [wiki](https://github.com/Daveiano/weewx-wdc/wiki/Webcams-and-Externals-Page)) GH-167

  - **Attention:** You need to update your skin.conf if you were using the webcams/externals:

    Before:

    ```
    [Extras]
       Include various external sources (eg. webcams) here.
       [[external_1]]
          source = '<img src="http://your-server.com/uploads/webcam01.jpg" />'
          title = Webcam 1
          title_long = "Webcam 1, facing North"
    ```

    After:

    ```
    [Extras]
       # Include various external sources (eg. webcams) here.
       [[externals]]
       #  description = 'This description will be shown at the Webcams/Externals page.
          [[[external_1]]]
             source = '<img src="http://your-server.com/uploads/webcam01.jpg" />'
             title = Webcam 1
             title_long = "Webcam 1, facing North"
             description = 'The image is updated every 80 seconds, nighttime every 100 seconds.'
             show_on_front = True
             show_on_page = True
    ```

## Changes made to skin.conf since 3.2.0

See https://github.com/Daveiano/weewx-wdc/compare/v3.2.0...11eed6b3#diff-ba225fb627dcbf577d0d1fde0f18d93e8e2cf2097c37086cd28ef23e8e7bd820 for a complete diff.

## Known Issues

- GH-181 [BUG] xAxis: Date mismatch in combined charts. Especially when using cumulative aggregation. Feel free to comment if you have more infos, or also nooticed this issue.

**Full Changelog**: https://github.com/Daveiano/weewx-wdc/compare/v3.2.0...v3.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Daveiano/weewx-wdc/releases/tag/v3.3.0'>v3.3.0</a>.</em>

**Comment by JonFear** [comment]
Awesome Dave, I will do the install later when things are a little quieter weather and domestic duties wise!

Thank you.

Jon
